pyinjective/composer.py

```diff
@@ -46,61 +50,256 @@
 from pyinjective.proto.injective.wasmx.v1 import tx_pb2 as wasmx_tx_pb
 from pyinjective.utils.denom import Denom
 
+# fmt: off
 REQUEST_TO_RESPONSE_TYPE_MAP = {
-    "MsgCreateSpotLimitOrder": injective_exchange_tx_pb.MsgCreateSpotLimitOrderResponse,
-    "MsgCreateSpotMarketOrder": injective_exchange_tx_pb.MsgCreateSpotMarketOrderResponse,
-    "MsgCreateDerivativeLimitOrder": injective_exchange_tx_pb.MsgCreateDerivativeLimitOrderResponse,
-    "MsgCreateDerivativeMarketOrder": injective_exchange_tx_pb.MsgCreateDerivativeMarketOrderResponse,
-    "MsgCancelSpotOrder": injective_exchange_tx_pb.MsgCancelSpotOrderResponse,
-    "MsgCancelDerivativeOrder": injective_exchange_tx_pb.MsgCancelDerivativeOrderResponse,
-    "MsgBatchCancelSpotOrders": injective_exchange_tx_pb.MsgBatchCancelSpotOrdersResponse,
-    "MsgBatchCancelDerivativeOrders": injective_exchange_tx_pb.MsgBatchCancelDerivativeOrdersResponse,
-    "MsgBatchCreateSpotLimitOrders": injective_exchange_tx_pb.MsgBatchCreateSpotLimitOrdersResponse,
-    "MsgBatchCreateDerivativeLimitOrders": injective_exchange_tx_pb.MsgBatchCreateDerivativeLimitOrdersResponse,
-    "MsgBatchUpdateOrders": injective_exchange_tx_pb.MsgBatchUpdateOrdersResponse,
-    "MsgDeposit": injective_exchange_tx_pb.MsgDepositResponse,
-    "MsgWithdraw": injective_exchange_tx_pb.MsgWithdrawResponse,
-    "MsgSubaccountTransfer": injective_exchange_tx_pb.MsgSubaccountTransferResponse,
-    "MsgLiquidatePosition": injective_exchange_tx_pb.MsgLiquidatePositionResponse,
-    "MsgIncreasePositionMargin": injective_exchange_tx_pb.MsgIncreasePositionMarginResponse,
-    "MsgCreateBinaryOptionsLimitOrder": injective_exchange_tx_pb.MsgCreateBinaryOptionsLimitOrderResponse,
-    "MsgCreateBinaryOptionsMarketOrder": injective_exchange_tx_pb.MsgCreateBinaryOptionsMarketOrderResponse,
-    "MsgCancelBinaryOptionsOrder": injective_exchange_tx_pb.MsgCancelBinaryOptionsOrderResponse,
-    "MsgAdminUpdateBinaryOptionsMarket": injective_exchange_tx_pb.MsgAdminUpdateBinaryOptionsMarketResponse,
-    "MsgInstantBinaryOptionsMarketLaunch": injective_exchange_tx_pb.MsgInstantBinaryOptionsMarketLaunchResponse,
+    "MsgCreateSpotLimitOrder":
```

Comment:
what if we use a workaround like that instead of full hardcoding: 
```
def get_response_type(request_name):
    response_type_name = f"{request_name}Response"
    
    return getattr(injective_exchange_tx_v2_pb, response_type_name, None)
```

Reply:
What happens with requests that have "Request" as the last word in the name but the response name is not "RequestResponse"?